kythe/javatests/com/google/devtools/kythe/analyzers/java/testdata/pkg/AnnotationComments.java

```diff
@@ -1,13 +1,14 @@
 package pkg;
 
-// Tests for comments that should be ignored.
+//- @+3AnnotationComments defines/binding AnnotationComments
+
+@Deprecated // TODO(#3459): This should not annotate the class, but does.
 public class AnnotationComments {
+  //- !{ _DeprecatedDoc documents AnnotationComments }
```

Comment:
Why use _DeprecatedDoc instead of _? We want to say this class has no documentation, not just that the deprecated comment isn't documentation (though that means the same thing in this case).

Reply:
sure sounds good